### PR TITLE
Add iq to metamask

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
fixes https://github.com/EveripediaNetwork/issues/issues/108

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
@kesar kindly review.

File: src/components/Layout/Token/TokenDetailsMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        {
+          method: 'metamask_watchAsset',
+          params: {
+            type: 'ERC20',
+            options: {
+              address: config.iqAddress,
+              symbol: tokenSymbol,
+              decimals: 18,
+              image:
+                'https://pbs.twimg.com/profile_images/1414736076158033921/nResATsF_400x400.png',
```
Comment (kesar) [inline src/components/Layout/Token/TokenDetailsMenu.tsx:25]: this is better if we self-hosted it in our local

File: src/components/Layout/Token/TokenDetailsMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            },
+          },
+          id: Math.round(Math.random() * 100000),
```
Comment (kesar) [inline src/components/Layout/Token/TokenDetailsMenu.tsx:28]: is this id needed? and it has to be randomized?

File: src/components/Layout/Token/TokenDetailsMenu.tsx (hunk 1 above)
Comment (Emmanueldmlr) [reply src/components/Layout/Token/TokenDetailsMenu.tsx:25]: Okay, I will switch it to local then.

File: src/components/Layout/Token/TokenDetailsMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    }
+  }
+
+  return (
+    <Menu placement="left-start">
+      <MenuButton>
+        <RiMore2Fill color="color" fontSize="20" fontWeight="bold" />
+      </MenuButton>
+      <MenuList onClick={() => handleAddTokenToMetamask(token)} boxShadow="xl">
+        <MenuItem>Add {token} token to Metamask</MenuItem>
```
Comment (kesar) [inline src/components/Layout/Token/TokenDetailsMenu.tsx:43]: I think we need a whitelist of tokens here. ETH doesnt work for example, so probably we should hide the 3 dots menu from the ones that can't be added 

I would just have this for IQ for now 👍🏻

**Comment by kesar** [comment]
functionality tested on IQ works fine 👍🏻 just some minor suggestions

File: src/components/Layout/Token/TokenDetailsMenu.tsx (hunk 2 above)
Comment (Emmanueldmlr) [reply src/components/Layout/Token/TokenDetailsMenu.tsx:28]: It's Optional. I think I should remove it then?

Comment (kesar) [reply src/components/Layout/Token/TokenDetailsMenu.tsx:28]: yeah, if it works without id, probably its not worth to add it (unless we know the reason behind 👍🏻 )

File: src/components/Layout/Token/TokenDetailsMenu.tsx (hunk 3 above)
Comment (Emmanueldmlr) [reply src/components/Layout/Token/TokenDetailsMenu.tsx:43]: Okay. I will do that now. Thanks for the review.

File: src/components/Layout/Token/TokenDetailsMenu.tsx (hunk 1 above)
Comment (Emmanueldmlr) [reply src/components/Layout/Token/TokenDetailsMenu.tsx:25]: Let me try it out

File: src/components/Layout/Token/TokenDetailsMenu.tsx (hunk 3 above)
Comment (Emmanueldmlr) [reply src/components/Layout/Token/TokenDetailsMenu.tsx:43]: @kesar, hiding the 3dots will affect the UI. so, instead, I disabled it (3 dots) when the token is not part of the whitelist of tokens.

Comment (kesar) [reply src/components/Layout/Token/TokenDetailsMenu.tsx:43]: sounds good 👍🏻

**Pull request merged**